File: university/3593-prgramming-languages/c/primes/primes.c
#include <stdio.h>
#include <stdlib.h>

bool is_prime(int n) {
    if (n <= 1 || (n % 2 == 0 && n != 2)) return false;
    for (int i = 3; i * i <= n; i += 2) {
        if (n % i == 0) return false;
    }
    return true;
}

int main() {
    for (int c = 0, n = 2; c < 10000; n++) {
        if (is_prime(n)) {
            printf("%d\n", n);
            c++;
        }
    }
}

File: university/3593-prgramming-languages/c/primes/primes.py
def is_prime(n):
    if n <= 1 or (n % 2 == 0 and n != 2):
        return False
    for i in range(3, int(n ** 0.5) + 1, 2):
        if n % i == 0:
            return False
    return True

c, n = 0, 2
while c < 10000:
    if is_prime(n):
        c += 1
    n += 1    
    